tests/src/test/java/alluxio/client/cli/fs/AbstractFileSystemShellTest.java

```diff
@@ -61,21 +56,11 @@
   protected JobMaster mJobMaster;
   protected LocalAlluxioJobCluster mLocalAlluxioJobCluster = null;
   protected JobShell mJobShell = null;
-  protected Map<NetworkAddressUtils.ServiceType, ServerSocket> mServiceMapping;
-
-  @Override
-  protected void customizeLocalAlluxioCluster(LocalAlluxioClusterResource.Builder resource) {
-    mServiceMapping = IntegrationTestUtils.createMasterServiceMapping();
-    resource.setSockets(mServiceMapping.get(NetworkAddressUtils.ServiceType.MASTER_RPC),
-        mServiceMapping.get(NetworkAddressUtils.ServiceType.MASTER_WEB));
-  }
 
   @Before
   public final void before() throws Exception {
     mLocalAlluxioCluster = mLocalAlluxioClusterResource.get();
-    mLocalAlluxioJobCluster = new LocalAlluxioJobCluster(
-        mServiceMapping.get(NetworkAddressUtils.ServiceType.JOB_MASTER_RPC),
-        mServiceMapping.get(NetworkAddressUtils.ServiceType.JOB_MASTER_WEB));
+    mLocalAlluxioJobCluster = new LocalAlluxioJobCluster();
```

Comment:
This isn't called until the `LocalAlluxioClusterResource` has been started. The `JOB_MASTER_RPC` and `WEB` ports need to be set before we get to this point